From 1828d3718659c1d35088ed7ff4df27b3c8af0367 Mon Sep 17 00:00:00 2001
From: Gergely Imreh <imrehg@gmail.com>
Date: Sun, 11 Apr 2021 18:40:27 +0000
Subject: [PATCH] switch base image to arm version

---
 Dockerfile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Dockerfile b/Dockerfile
index 2090d27..fc56d0b 100644
--- a/Dockerfile
+++ b/Dockerfile
@@ -1,3 +1,3 @@
-FROM atdr.meo.ws/archiveteam/grab-base
+FROM imrehg/archiveteam-arm-grab-base
 COPY . /grab
 RUN ln -fs /usr/local/bin/wget-lua /grab/wget-at
-- 
2.31.1

